pybamm/parameters/bpx.py

```diff
@@ -0,0 +1,377 @@
+from bpx import BPX, Function, InterpolatedTable
```

Comment:
this should be a private file with private function names (all start with underscore). I know we haven't been good about this in the past but better late than never (see #2427 )